tools/provision.sh

```diff
@@ -143,17 +143,17 @@ else # AUTH_METHOD_NONE
 
     # RHEL nightly repos need to be overridden
     case "${ID}-${VERSION_ID}" in
-        "rhel-8.6")
-            # Override old rhel-8.json and rhel-8-beta.json because RHEL 8.6 test needs nightly repos
-            sudo cp /usr/share/tests/osbuild-composer/repositories/rhel-86.json "$REPODIR/rhel-8.json"
+        "rhel-8.7")
+            # Override old rhel-8.json and rhel-8-beta.json because RHEL 8.7 test needs nightly repos
+            sudo cp /usr/share/tests/osbuild-composer/repositories/rhel-87.json "$REPODIR/rhel-8.json"
```

Comment:
Does this still need to be `rhel-8.json`? I think we want to use `rhel-87.json` now, but I'd have to check to be sure...

Reply:
I'm not sure. Probably not. @ondrejbudai can you help us here?

Reply:
I think it should be `rhel-87.json`. @thozza?

Reply:
Yes, this should be `rhel-87.json`.

Unfortunately, we are still matching the full distro name as defined in the distro definition against the file name. So `rhel-8.json` would be ever used only for RHEL-8.3.